src/interfaces.ts

```diff
@@ -1,5 +1,5 @@
 export type ChargifyId = number;
-export type ChargifyDate = string | null;
+export type ChargifyDate = string | null; // example: "2019-01-30T11:56:05-08:00"
```

Comment:
does tz matter? utc?

Reply:
The timezone is whatever Chargify decides to use. All the of the date fields are reported from Chargify to us, so we shouldn't need to worry about it.